Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -34,6 +34,9 @@ public protocol Environmenting: AnyObject {
 
     /// Returns the path to the directory where the async queue events are persisted.
     var queueDirectory: AbsolutePath { get }
+    
+    /// Returns true if the environment is ARM.
+    var isArm: Bool { get }
```

Comment:
where will be this new property used? Or is this just laying some groundwork to get the M1 support going?

Reply:
I don't think we'll need to use it in the end. The only thing we'll have to do is extending our release logic to build the binaries and the libraries to include both architectures, x86 and arm64. I'd still leave the method here in case we need it in the future.